### PR TITLE
Fix #5406: hints overflow

### LINKED ISSUE #5406: Info of parameterHintsWidget overflows
### Description
image

**OS and Theia version:**
Theia v0.7.0

### Suggestion
now:
```css
.monaco-editor .parameter-hints-widget>.wrapper {
  max-width: 440px;
  display: flex;
  flex-direction: column;
 }
```

fix:
```
.monaco-editor .parameter-hints-widget>.wrapper {
  max-width: 440px;
  display: flex;
  flex-direction: column;
  overflow: hidden;
}
```

### PR DESCRIPTION
**Pull request by AlbertXiao1994**
Fix #5406: hints overflow

Fix a error when parameterHintsWidget contains long info

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
Thank you for your contribution @AlbertXiao1994, please make sure to sign the Eclipse [ECA](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git) (reason for the failed CI check).

File: packages/monaco/src/browser/style/index.css [hunk 1]
```diff
@@ -63,3 +63,7 @@
     border-bottom-color: var(--theia-accent-color2);
 	position: relative;
 }
+
+.monaco-editor .parameter-hints-widget>.wrapper {
```
Comment (vince-fugnitto) [inline packages/monaco/src/browser/style/index.css]: [minor] can you adjust the spacing so it becomes:

```css
.monaco-editor .parameter-hints-widget > .wrapper
```

**Comment by vince-fugnitto** [comment]
It works great! 
Please make sure to address the comments, and I'll approve and merge :)

File: packages/monaco/src/browser/style/index.css (hunk 1 above)
Comment (AlbertXiao1994) [reply packages/monaco/src/browser/style/index.css]: I did it as you said

**Comment by vince-fugnitto** [comment]
I'll have to wait for you to sign the [Eclipse Contributors Agreement](https://www.eclipse.org/legal/ECA.php) before I can proceed unfortunately.

**Comment by AlbertXiao1994** [comment]
> I'll have to wait for you to sign the [Eclipse Contributors Agreement](https://www.eclipse.org/legal/ECA.php) before I can proceed unfortunately.

I have signed it

**Comment by vince-fugnitto** [comment]
Sorry to bother you again, do you mind squashing your commits into a single commit with the desired change? In Theia, we use `rebase and merge` when merging the PR.

**Pull request closed without merging**